app/views/prison/dashboards/_request_details_row.html.erb

```diff
@@ -1,24 +1,17 @@
 <tr class="hidden-row hide" aria-hidden="true" aria-expanded="false">
   <td colspan="9" class="request-details">
     <div class="request-row-details">
-      <% if visit.slot_granted? -%>
+      <% visit.slots.each_with_index do |slot, i| %>
```

Comment:
`ConcreteSlot` is not an ActiveModel so doesn't implement `#to_partial_path` for this to work. I'm tempted to leave it as it is as the partial won't be reused anywhere else. What do you think?